src/System.Private.ServiceModel/src/System/IdentityModel/Selectors/X509CertificateValidator.cs

```diff
@@ -108,6 +208,44 @@ private static string GetChainStatusInformation(X509ChainStatus[] chainStatus)
                 return String.Empty;
             }
         }
+
+
+        class PeerOrChainTrustValidator : X509CertificateValidator
+        {
+            X509CertificateValidator chain;
+            PeerTrustValidator peer;
```

Comment:
NIT: style, should be `_chain` and `_peer`. Also remove `this.` from blocks below